packages/flutter/lib/src/services/text_input.dart

```diff
@@ -2226,7 +2244,9 @@ class _PlatformTextInputControl with TextInputControl {
   Map<String, dynamic> _configurationToJson(TextInputConfiguration configuration) {
     final Map<String, dynamic> json = configuration.toJson();
     if (TextInput._instance._currentControl != _PlatformTextInputControl.instance) {
-      json['inputType'] = TextInputType.none.toJson();
+      json['inputType'] = TextInputType.noneWithOptions(
+        forceMultiline: configuration.inputType == TextInputType.multiline,
```

Comment:
From what I understand we should still continue to send `TextInputType.none` in both the multi-line case and the single-line case of custom text input controls because that prevents the keyboard from being shown. Won't this make the keyboard show again?

Reply:
Maybe the `forceMultiline` flag can be processed by the web embedder to decide whether to use textarea or input.

Reply:
@Renzo-Olivares Yes, this won't show the keyboard again. This still sends `TextInputType.none` to the engine. And also, send `forceMultiline` stated in `TextInputType.none`.

Reply:
I have updated the [PR](https://github.com/flutter/engine/pull/45522) in the web engine to handle `forceMultiline `. But have not reviewed it yet.

Reply:
I have updated the [PR](https://github.com/flutter/engine/pull/45522) in the web engine to handle `forceMultiline `. But have not reviewed it yet.